internals/src/macros.rs

```diff
@@ -161,34 +161,36 @@ macro_rules! const_assert {
 /// # use core::fmt::{Display, Debug};
```

Comment:
```suggestion
/// #![allow(unused)]
/// # use core::fmt::{Display, Debug};
```

We can just allow unused at the top of this example and not worry about all the other changes.

Reply:
Done.
Using an inner attribute gives `error: an inner attribute is not permitted in this context`, so I put the example code inside a function block and used an outer attribute on it.

Reply:
Oh I put it in the wrong place, an inner attribute has to go at the top line. I did run it locally so it works.